src/components/ArticleContainer/ArticleContainer.jsx

```diff
@@ -0,0 +1,28 @@
+import { getAllArticles } from "../../utils/api";
+import { useEffect, useState } from "react";
+import "./ArticleContainer.css";
+import ArticleCard from "./ArticleCard/ArticleCard";
+
+function ArticleContainer() {
+  const [articles, setArticles] = useState([]);
+
+  useEffect(() => {
+    getAllArticles().then((data) => {
+      setArticles(data.articles);
+    });
+  }, []);
```

Comment:
I'd highly recommend a loading pattern here as while this is spinning up there's nothing stopping the map from attempting to render articles that don't exist which will cause errors.